test/integration.sh

```diff
@@ -222,6 +222,17 @@ else
   exit 1
 fi
 
+echo "> test the LOG_LEVEL command"
+export CL_LOG_LEVEL=Debug
```

Comment:
I'm not sure that this is relevant for a test, it should use file configuration, don't you think ?